docker-compose.test.yml

```diff
@@ -8,6 +8,8 @@ services:
       context: .
       dockerfile: Dockerfile
       target: test
+      args:
+        - CI=1
```

Comment:
I'm surprised we need to set this and CircleCI doesn't set this when appropriate on its own?